doc/source/text.rst

```diff
@@ -429,6 +429,27 @@ String ``Index`` also supports ``get_dummies`` which returns a ``MultiIndex``.
 
 See also :func:`~pandas.get_dummies`.
 
```

Comment:
need a reference tag here

Reply:
I don't understand what should be added where. Reference to what? Where - In the "concatenation" section I'm writing?